src/pages/error-page/errorPage.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect } from 'react';
```

Comment:
Please, rename the file to the `error-page`, we name components using lowercase and dashes

Reply:
Done